### ISSUE TITLE
Release 4.8.1

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1405]
- [x] Address all coding and style violations detected during a *Release* build (send a regular pull request which must be merged before continuing)
- [x] ensure completeness of draft [release notes](https://github.com/FakeItEasy/FakeItEasy/releases), including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
- [x] run `tools\prepare_release.cmd x.y.z` to:
    - change "vNext" names to appropriate x.y.z version for the next GitHub Release, milestone, and release checklist (this issue)
    - create a new draft GitHub Release, milestone, and release checklist
    - create release/x.y.z branch on upstream, branching from develop
    - submit a pull request to merge the release/x.y.z branch to master
- [x] merge the pull request, _which may be done by the PR creator, unlike a regular pull request_
- [x] tag master with the new version number and push to upstream. This will cause the build server to
    - push the nupkgs to [nuget.org](https://www.nuget.org/profiles/FakeItEasy) and
    - attach all nupkgs from the tagged build to the GitHub Release.
- [x] wait for NuGet package validation
- [x] de-list pre-release NuGet packages if present
- [x] publish the GitHub release
- [x] tweet, mentioning contributors, and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [ ] close this issue
- [ ] close the [milestone](https://github.com/FakeItEasy/FakeItEasy/milestones?state=open) for this release

**Comment by blairconrad** [comment]
@thomaslevesque, shall we release? We have only 1 bug fix, but it's a good way to test the build scripts and prepare-release scripts in a low-pressure environment.

**Comment by blairconrad** [comment]
Oh! It's Friday. Mind like a sieve. I could do on Tuesday.

**Comment by thomaslevesque** [comment]
> @thomaslevesque, shall we release? We have only 1 bug fix, but it's a good way to test the build scripts and prepare-release scripts in a low-pressure environment.

4.8.1 then? Since it's just a bug fix

**Comment by blairconrad** [comment]
Yup.

**Comment by blairconrad** [comment]
```
Get FakeItEasy 4.8.1!
- Fixed inability to step into source of .NET 4.5 version of the library

Thanks to @adamralph for upgrading build tooling
https://github.com/FakeItEasy/FakeItEasy/releases/4.8.1
```

**Comment by blairconrad** [comment]
Automatic deployment failed because we can't find the nuget.exe. Created #1430 and uploading packages to nuget manually.

**Comment by blairconrad** [comment]
https://twitter.com/fakeiteasyfx/status/1036924099468124160

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 4.8.1](https://github.com/FakeItEasy/FakeItEasy/releases/4.8.1).
